gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -126,6 +127,7 @@ When running in Kubernetes, you are responsible for:
 
 * Adapting the base Helm chart for your specific needs
 * Managing the Kubernetes cluster, including:
+//TODO: "ingress controller" ok?
```

Comment:
@seblaz What about "Ingress controller" below?

Reply:
I suggest changing "Managing the Kubernetes cluster, including:" to "Managing the Kubernetes Deployment, including:"

Reply:
For "Ingress controller and external load balancing" I suggest leaving the ingress controller part out of the line, since it's already included in the "deployment"

Reply:
done